### ISSUE TITLE
Request: Sync settings when a player logs in

### ISSUE THREAD

**Issue by dmrickey** [BlitzKraig/fvtt-SocketSettings#4]
I've got a couple mods that have some backwards default options (mainly because I'm opinionated and also because they implemented better functionality after the mod had been released for a while and didn't want to switch the default to the "better" option out from under the users).

But basically there are a few specific settings that I would always like to sync whenever a player logs in. I'd rather not have to remember to do it myself and some of my players clear their cache, log in from different computers, or I'll just play with new people so I often have to sync these options when the players log in.

Basically a new button the currently existing button that's labeled something like "sync whenever player logs in" (or probably something less verbose).
